app/controllers/api/v1/employees_controller.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # Employees API controller
+    class EmployeesController < ApplicationController
+      before_action :authenticate_user!
+      before_action :set_company
+      before_action :set_member, only: %i[show update destroy]
+      before_action :validate_role, only: %i[create update]
+      rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+
+      def index
+        if manager?
+          @members = @company.members
+        else
+          handle_unauthorized
+        end
+      end
+
+      def show
+        if @member.managed_by?(current_user)
+          respond_to { |format| format.json { render :show } }
+        else
+          handle_unauthorized
+        end
+      end
+
+      def current
```

Comment:
what is the reason to have this action?

Reply:
I was choosing between passing User-ID as SPA root_tag attribute and getting User by API. Is the root_tag attribute ok?

Reply:
what is root tag? not sure that I imagine the same what you mean :)

Reply:
for example:
```
<body>
  <div id="app" uid="2">
  </div>
</body>
```
tag named "app" is the root_tag(container) for SPA, uid => User ID

Reply:
we can get user_id from current user :)

Reply:
of course :) but, how can the SPA-client get it?

Reply:
not sure why do we need it hardcoded? if each request for authorized user goes to controller where we know user